### PR TITLE
Update gam report usageparameters to account for Google API change

### PR DESCRIPTION
**Pull request by taers232c**

### CODE REVIEW

**Comment by jay0lee** [comment]
Can we reuse the logic in ```_check_full_data_available(fullDataRequired='all')``` here?

**Comment by taers232c** [comment]
I'll check it out

File: src/gam/gapi/reports.py [hunk 1]
```diff
@@ -78,36 +76,29 @@ def showUsageParameters():
...
-                               one_day).strftime(YYYYMMDD_FORMAT)
+            result = gapi.call(endpoint,
+                               'get',
+                               throw_reasons=throw_reasons,
+                               date=tryDate,
+                               customerId=customerId,
+                               fields='warnings,usageReports',
+                               **kwargs)
+            warnings = result.get('warnings', [])
+            fullDataRequired = ['all']
```
Comment (jay0lee) [inline src/gam/gapi/reports.py:87]: this can be moved out of the ```while True``` loop. Otherwise looks good. Thanks.

**Comment by jay0lee** [comment]
There's a chance that: a new parameter is added and starts showing on reports on a recent day that still only has partial results. So our loop should always add any parameters we get to all_parameters even if the results are partial. Then we keep adding any found parameters from recent days but we ensure our list of all_parameters is complete by looping until we find a date in the past which has no warnings / partial results.

**Pull request merged**